src/components/UsersActions/UsersActionsRemoveFromOrg.tsx

```diff
@@ -28,23 +28,26 @@ function FormDefault({
   const isPending = actionStatus === 'pending';
   return (
     <>
-      <p id={modalHeadingId(user)}>
-        Are you sure you want to remove <strong>{user.username}</strong> from
-        this organization?
+      <p id={modalHeadingId(user)} className="font-sans-md text-bold">
+        Are you sure you want to remove {user.username} from this organization?
+      </p>
+      <p>
+        If you remove this account, you’ll have to add it back to this
+        organization in order to restore its access.
       </p>
       {isPending && <p>Removal pending...</p>}
       <div className="usa-modal__footer">
         <form onSubmit={onSubmit}>
           <Button type="submit" disabled={isPending}>
-            Remove
+            Yes, remove the account
           </Button>{' '}
           <Button
             unstyled
             type="button"
             onClick={onCancel}
             disabled={isPending}
           >
-            Cancel
+            Go back
```

Comment:
When the modal’s closed, can we return focus to the element/button that opened the modal? (It’s extremely good for a11y: https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html, https://docs.deque.com/issue-help/1.0.0/en/focus-modal-not-returned)

Reply:
Great catch—done!

Reply:
Works _perfectly_.